backend/runners/load_census_tract_demographics.py

```diff
@@ -1,75 +1,130 @@
-"""Methods to upload demographic data to the Postgres database."""
```

Comment:
Should we keep both import files?
Or have a toogle / import from 100644 / 100755?

Reply:
100644 / 100755? 

I think we should not use the Kaggle data at all, good point I meant to remove it.